CHANGELOG.md

```diff
@@ -3,11 +3,16 @@
 # Changelog
 
 - [Changelog](#changelog)
-  - [3.10.0](#3100)
+  - [3.11.0](#3110)
     - [New Features](#new-features)
     - [Bug Fixes](#bug-fixes)
   - [Documentation](#documentation)
   - [Development](#development)
+  - [3.10.0](#3100)
+    - [New Features](#new-features-1)
+    - [Bug Fixes](#bug-fixes-1)
+  - [Documentation](#documentation-1)
+  - [Development](#development-1)
```

Comment:
I sooo didn't catch this last time around - the indentation is bad :sweat_smile: